src/scripts/main.js

```diff
@@ -1,3 +1,306 @@
+
+const nameInput = document.createElement('input');
+
+nameInput.setAttribute('name', 'name');
+nameInput.setAttribute('type', 'text');
+nameInput.setAttribute('data-qa', 'name');
+nameInput.required = true;
+nameLabel.appendChild(nameInput);
+form.appendChild(nameLabel);
+
+// Position
+const positionLabel = document.createElement('label');
+
+positionLabel.textContent = 'Position: ';
+
+const positionInput = document.createElement('input');
+
+positionInput.setAttribute('name', 'position');
+positionInput.setAttribute('type', 'text');
+positionInput.setAttribute('data-qa', 'position');
+positionInput.required = true;
+positionLabel.appendChild(positionInput);
+form.appendChild(positionLabel);
+
+// Office
+const officeLabel = document.createElement('label');
+
+officeLabel.textContent = 'Office: ';
+
+const officeSelect = document.createElement('select');
+
+officeSelect.setAttribute('name', 'office');
+officeSelect.setAttribute('data-qa', 'office');
+officeSelect.required = true;
+
+const offices = [
+  'Tokyo',
+  'Singapore',
+  'London',
+  'New York',
+  'Edinburgh',
+  'San Francisco'];
+
+offices.forEach(office => {
+  const option = document.createElement('option');
+
+  option.value = office;
+  option.textContent = office;
+  officeSelect.appendChild(option);
+});
+
+officeLabel.appendChild(officeSelect);
+form.appendChild(officeLabel);
+
+// Age
+const ageLabel = document.createElement('label');
+
+ageLabel.textContent = 'Age: ';
+
+const ageInput = document.createElement('input');
+
+ageInput.setAttribute('name', 'age');
+ageInput.setAttribute('type', 'number');
+ageInput.setAttribute('data-qa', 'age');
+ageInput.required = true;
+ageLabel.appendChild(ageInput);
+form.appendChild(ageLabel);
+
+// Salary
+const salaryLabel = document.createElement('label');
+
+salaryLabel.textContent = 'Salary: ';
+
+const salaryInput = document.createElement('input');
+
+salaryInput.setAttribute('name', 'salary');
+salaryInput.setAttribute('type', 'number');
+salaryInput.setAttribute('data-qa', 'salary');
+salaryInput.required = true;
+salaryLabel.appendChild(salaryInput);
+form.appendChild(salaryLabel);
+
+// Button
+const submitButton = document.createElement('button');
+
+submitButton.setAttribute('type', 'submit');
+submitButton.textContent = 'Save to table';
+form.appendChild(submitButton);
+
+// Add form submit event listener
+form.addEventListener('submit', function(e) {
+  e.preventDefault();
+
+  // Get form data
+  const person = nameInput.value.trim();
+  const position = positionInput.value.trim();
+  const office = officeSelect.value;
+  const age = parseInt(ageInput.value);
+  const salary = parseFloat(salaryInput.value);
+
+  // error - no new row
+
+  if (person.length < 4) {
+    pushNotification(450, 10, 'Error!',
+      'Name must have minimum 4 lettes', 'error');
+
+    return;
+  }
+
+  if (position.length === 0) {
+    pushNotification(450, 10, 'Error!',
+      'Name must have minimum 4 lettes', 'error');
```

Comment:
Update error message to 'Position must have at least 1 character' since it's related to position length.